### PR TITLE
chore: [#184198991] re-add zebra pattern to signers section

### PR DESCRIPTION
**Pull request by AnJuHyppolite**
## Description

Re-add zebra pattern to signers section and a line to the bottom of the table once the BOD and signers sections are complete.

### Ticket

[#184198991](https://www.pivotaltracker.com/story/show/184198991)

## Code author checklist

- [x] I have performed a self-review of my code
- [x] I have created and/or updated relevant documentation, if necessary
- [x] I have not used any relative imports
- [x] I have checked for and removed instances of unused config from CMS
- [x] I have pruned any instances of unused code
- [x] I have not added any markdown to labels, titles and button text in config

### CODE REVIEW

**Comment by aloverso** [comment]
@AnJuHyppolite why change from the CSS-based pattern to doing it in JS? As I was scrolling through, I was going to leave a comment letting you know that this could be done using CSS nth-child. But it looks like we already had that, and are switching to calculating in javascript instead - how come?

File: web/src/components/tasks/business-formation/contacts/Addresses.tsx [hunk 1]
```diff
@@ -121,7 +121,11 @@ export const Addresses = <T extends FormationMember | FormationIncorporator>(
           props.addressData.map((it, index) => {
             return (
               <Fragment key={index}>
-                <tr className="margin-bottom-1" key={index} data-testid={`${props.fieldName}-${index}`}>
+                <tr
+                  className={`margin-bottom-1 ${index % 2 === 1 ? "bg-base-lightest" : ""}`}
```
Comment (adidner) [inline web/src/components/tasks/business-formation/contacts/Addresses.tsx:125]: [sand] so it seems like we reuse this code many times `${index % 2 === 1 ? "bg-base-lightest" : ""}`

so we could probably make this a function `const zerbraOnOdd = (index) =>  index % 2 === 1 ? "bg-base-lightest" : ""`

then use it like so `className={`margin-bottom-1 ${zebraOnOdd(index)}`}`

**Comment by AnJuHyppolite** [comment]
> @AnJuHyppolite why change from the CSS-based pattern to doing it in JS? As I was scrolling through, I was going to leave a comment letting you know that this could be done using CSS nth-child. But it looks like we already had that, and are switching to calculating in javascript instead - how come?

@aloverso: Because the error message is rendered as a row, the CSS-based pattern, error messages were that are even-number-indexed are being highlighted. If you have another suggestion, I'd like to try it.

**Comment by adidner** [comment]
> > @AnJuHyppolite why change from the CSS-based pattern to doing it in JS? As I was scrolling through, I was going to leave a comment letting you know that this could be done using CSS nth-child. But it looks like we already had that, and are switching to calculating in javascript instead - how come?
> 
> @aloverso: Because the error message is rendered as a row, the CSS-based pattern, error messages were that are even-number-indexed are being highlighted. If you have another suggestion, I'd like to try it.

TLDR is that we're using <Table> which is having issues looking like the design and accommodating that CSS, should probably refactor to use grid or flex box which would allow us to do the CSS instead but I think this is also fine

**Comment by aloverso** [comment]
> > @AnJuHyppolite why change from the CSS-based pattern to doing it in JS? As I was scrolling through, I was going to leave a comment letting you know that this could be done using CSS nth-child. But it looks like we already had that, and are switching to calculating in javascript instead - how come?
> 
> @aloverso: Because the error message is rendered as a row, the CSS-based pattern, error messages were that are even-number-indexed are being highlighted. If you have another suggestion, I'd like to try it.

@AnJuHyppolite oh I see, got it. Would it work to add a class name instead? Like, for any non-error row that should receive the zebra highlighting, you can just give it a className like `<tr className="zebra-highlighting">`. And then change the CSS to apply the nth-child styling to instances of that class, instead of instances of `tr`?

File: web/src/styles/base/background-colors.scss [hunk 2]
```diff
@@ -204,3 +204,6 @@
 .bg-accent-semi-cool {
   background-color: $accent-semi-cool !important;
 }
+.bg-base-lightest {
```
Comment (FarazA22) [inline web/src/styles/base/background-colors.scss:207]: [Pebble] Some of our colors are already available through USWDS/NJ's CSS file we import. I believe this can be removed.
